===================================================================
RCS file: /cvs/src/src/newlib/libc/include/sys/unistd.h,v
retrieving revision 1.77
retrieving revision 1.78
diff -u -r1.77 -r1.78
--- src/newlib/libc/include/sys/unistd.h	2010/10/08 15:28:49	1.77
+++ src/newlib/libc/include/sys/unistd.h	2011/07/20 01:14:50	1.78
@@ -487,10 +487,11 @@
 #define _CS_XBS5_LPBIG_OFFBIG_LINTFLAGS       16
 #define _CS_POSIX_V7_WIDTH_RESTRICTED_ENVS    17
 #define _CS_POSIX_V6_WIDTH_RESTRICTED_ENVS    _CS_POSIX_V7_WIDTH_RESTRICTED_ENVS
+#define _CS_XBS5_WIDTH_RESTRICTED_ENVS        _CS_POSIX_V7_WIDTH_RESTRICTED_ENVS
 #define _CS_POSIX_V7_THREADS_CFLAGS           18
 #define _CS_POSIX_V7_THREADS_LDFLAGS          19
 #define _CS_V7_ENV                            20
-#define _CS_V6_ENV                           _CS_V6_ENV
+#define _CS_V6_ENV                            _CS_V7_ENV
 #endif
 
 #ifndef __CYGWIN__
